pom.xml

```diff
@@ -39,21 +39,55 @@
     <licenses>
         <license>
             <name>The Apache Software License, Version 2.0</name>
-            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <url>httpS://www.apache.org/licenses/LICENSE-2.0.txt</url>
             <distribution>repo</distribution>
         </license>
     </licenses>
 
-    <prerequisites>
-        <maven>3.5.0</maven>
-    </prerequisites>
+    <developers>
+      <developer>
+        <id>Apache Hugegraph(incubating)</id>
+        <email>dev-subscribe@hugegraph.apache.org</email>
+        <url>https://hugegraph.apache.org/</url>
+      </developer>
+    </developers>
+
+    <mailingLists>
+      <mailingList>
+        <name>Development List</name>
```

Comment:
Development Mailing List

Reply:
keep the origin is also fine, the xml tag is `mailingList` already, and other apache repo has the similar name:

<img width="538" alt="image" src="https://user-images.githubusercontent.com/17706099/206418991-e189e2ef-8a33-4751-9c2d-93268111ac8b.png">

<img width="471" alt="image" src="https://user-images.githubusercontent.com/17706099/206418084-bd9cd20d-ff1f-4a04-a43b-0c183fa89c56.png">